app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UserSerializer < ApplicationSerializer
+  attributes :id, :first_name, :last_name, :email, :phone_number
```

Comment:
Since you removed some columns from the user model as suggested here https://github.com/Wolfpack-Digital/rails-template/pull/20#discussion_r826301452, `:phone_number` is no longer present on a user.